crates/matrix-sdk-base/src/media.rs

```diff
@@ -124,10 +124,10 @@ impl MediaEventContent for StickerEventContent {
 
 impl MediaEventContent for AudioMessageEventContent {
```

Comment:
Can we get a test going that now checks for correctness here, preventing us from regressing here.

Reply:
I've added tests for all EventContent types which support any kind of encryption.